src/azure-cli/requirements.py3.Darwin.txt

```diff
@@ -71,7 +71,7 @@ azure-mgmt-redhatopenshift==0.1.0
 azure-mgmt-redis==7.0.0rc1
 azure-mgmt-relay==0.1.0
 azure-mgmt-reservations==0.6.0
-azure-mgmt-resource==12.1.0
+azure-mgmt-resource==16.1.0
```

Comment:
Because the Python SDK version you are using is the track 2 version, and the Python SDK of track 2 has many breaking changes.
Therefore, you need to wait for us to merge the code of track 2 migration before you can merge this PR. We plan to merge the code of track 2 migration in the next sprint. [PR link](https://github.com/Azure/azure-cli/pull/17783)
If this time doesn't meet your ETA, please consider using the python SDK of track 1.

Reply:
@detienne20 By the way, may I ask why do you need to upgrade the SDK